app/experimenter/nimbus-ui/src/components/TableSummary/index.tsx

```diff
@@ -6,155 +6,81 @@ import React from "react";
-                ? experiment.channels
-                    .map((expChannel) =>
-                      displayConfigLabelOrNotSet(
-                        "channel",
-                        expChannel,
-                        channels,
-                      ),
-                    )
-                    .join(", ")
-                : notSet("channel")}
-              {", "}
-            </span>
-            <span data-testid="experiment-ff-min">
-              {displayConfigLabelOrNotSet(
-                "Firefox minimum version",
-                experiment.firefoxMinVersion,
-                firefoxMinVersion,
-              )}
-            </span>
-            <span className="d-block">
-              <span data-testid="experiment-population">
-                {experiment.populationPercent
-                  ? `${experiment.populationPercent}% of population`
-                  : notSet("Population percentage")}
-              </span>
-              <span data-testid="experiment-total-enrolled">
-                {" "}
-                totalling{" "}
-                {experiment.totalEnrolledClients
-                  ? `${experiment.totalEnrolledClients.toLocaleString()} expected enrolled clients`
-                  : notSet("expected enrolled clients")}
-              </span>
-            </span>
-          </td>
+          <th>Hypothesis</th>
+          <td data-testid="experiment-hypothesis">{experiment.hypothesis}</td>
         </tr>
         <tr>
-          <th className="font-weight-bold">Duration</th>
-          <td>
-            <span data-testid="experiment-duration">
-              {displayDaysOrNotSet(
-                "Proposed duration",
-                experiment.proposedDuration,
-              )}{" "}
-            </span>
-            <span data-testid="experiment-enrollment">
-              over an enrollment period of{" "}
-              {displayDaysOrNotSet(
-                "proposed enrollment",
-                experiment.proposedEnrollment,
-              )}
-            </span>
+          <th>Public description</th>
+          <td data-testid="experiment-description">
+            {experiment.publicDescription ? (
+              experiment.publicDescription
+            ) : (
+              <NotSet />
+            )}
           </td>
         </tr>
+        {experiment.featureConfig?.name && (
+          <tr>
+            <th>Feature config</th>
+            <td data-testid="experiment-feature-config">
+              {experiment.featureConfig.name}
+            </td>
+          </tr>
+        )}
+        {experiment.primaryProbeSets?.length !== 0 && (
+          <tr>
+            <th>Primary probe sets</th>
+            <td data-testid="experiment-probe-primary">
+              {experiment
+                .primaryProbeSets!.map((probeSet) => probeSet?.name)
+                .join(", ")}
+            </td>
+          </tr>
+        )}
+        {experiment.secondaryProbeSets?.length !== 0 && (
```

Comment:
is `!== 0` neccesary? I think `.length` would evaluate to truthy

Reply:
It is, or `> 0`. Otherwise, this happens:

<img width="1183" alt="image" src="https://user-images.githubusercontent.com/13018240/101184086-e8a0d680-3615-11eb-973f-c27ea61c27aa.png">

I could fix this by using a ternary instead of `&&`, but I liked using includes with a more strict check instead.